onyo/cli/rm.py

```diff
@@ -51,18 +51,24 @@
 .. code:: shell
 
     $ onyo rm --message "Bob retired; he won at bingo" admin/Bingo\ Bob/
+
+Remove a building and all of its assets:
+
+.. code:: shell
+
+    $ onyo rm --recursive --message "The big fire..." buildings/20.3/
+
 """
 
 
 def rm(args: argparse.Namespace) -> None:
     r"""
     Delete **ASSET**\ s and/or **DIRECTORY**\ s.
 
-    Directories and asset directories are deleted along with their contents,
-    if the ``--recursive`` flag is set. Otherwise, fails on non-empty directories.
+    If the ``--recursive`` flag is set, directories are deleted along with their
```

Comment:
Not sure if this is a digression... but I'm reflecting on the writing style here.

What do you think about the following?

> When ``--recursive`` is used, directories are deleted along with their

Reply:
I personally prefere to start with "if" instead of "when" because the next sentence starts with an "otherwise". The rest of your suggestion I applied :)